src/ContaoManager/Plugin.php

```diff
@@ -62,7 +62,7 @@ public function getPackageDependencies()
      */
     public function registerContainerConfiguration(LoaderInterface $loader, array $managerConfig)
     {
-        $loader->load(__DIR__.'/../Resources/config/config.yml');
+        $loader->load(__DIR__.'/../../config/config.yaml');
```

Comment:
This change seems unrelated.

Reply:
see: https://github.com/m-vo/contao-facebook-import/pull/33